src/samplers/rezolus/stats.rs

```diff
@@ -0,0 +1,74 @@
+use crate::*;
+use metriken::metric;
+use metriken::Counter;
+use metriken::Gauge;
+use metriken::LazyCounter;
+
+#[metric(
+    name = "rezolus/cpu/usage/user",
+    description = "The amount of CPU time Rezolus was executing in user mode"
+)]
+pub static RU_UTIME: LazyCounter = LazyCounter::new(Counter::default);
+
+histogram!(
```

Comment:
Curious: is defining both of them separately the same as `counter_with_histogram!`?

Reply:
It is. I switched over to the convenience macros for all these stats. If we start using labels, we'll need to go with the previous approach to add that metadata.